### ISSUE TITLE
Turbolinks 5 and double animation

### ISSUE THREAD

**Issue by rept** [ankane/chartkick#390]
Hi,

I'm using Turbolinks 5.   I have this in my global options:
`    animation: {
        duration: 300,
        easing: "linear"
    }}`

The first time I completely load the page that contains the chart it load perfectly and shows the animation once.  When I click on the link in the nav bar that loads the page again, the animation shows immediately again (when the GET request is still pending) and when the GET finishes it animates it again.

In other words I see the animation twice.

Is anyone else experiencing this as well?  How can I fix this?

**Comment by klausbadelt** [comment]
I still have this issue.

**Comment by ankane** [comment]
For future readers, this is due to Turbolinks' preview feature. You can disable it with:

```html
<meta name="turbolinks-cache-control" content="no-preview">
```
